exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/trace_exporter/__init__.py

```diff
@@ -66,8 +65,8 @@ class OTLPSpanExporter(
 
     Args:
         endpoint: OpenTelemetry Collector receiver endpoint
-        insecure: Connection type
-        credentials: Credentials object for server authentication
+        credentials: Optional ChannelCredentials object for server authentication.
```

Comment:
Does an equivalent change need to be made to the metrics exporter in the metrics branch?

Reply:
Yes, will send in another PR.